srcpkgs/hugin/patches/exiv2-0.28.patch

```diff
@@ -0,0 +1,77 @@
+Patch-Source: https://gitlab.archlinux.org/archlinux/packaging/packages/hugin/-/blob/main/exiv2-0.28.patch
```

Comment:
Not sure, but wouldn't we need to bump a revision in the `hugin`'s template so that it will rebuild with this patch?

Reply:
Right, I think I did bump it, but Git has a habit of getting rid of revision bumps sometimes...

Reply:
and I think `libextractor` should have bumped revision as well

Reply:
Done, thanks!